src/System.Data.Common/src/System/DBNull.cs

```diff
@@ -1,31 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@stephentoub @weshaggard just confirming, do we want to use the DBNull from System.Private.CoreLib instead of this one?